.github/workflows/phar-creation.yml

```diff
@@ -0,0 +1,66 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Building Require-Checker phar"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - "master"
+
+jobs:
+    build:
+        name: "Building Require-Checker phar"
+
+        runs-on: ${{ matrix.operating-system }}
+
+        strategy:
+            matrix:
+                dependencies:
+                    - "locked"
+                php-version:
+                    - "7.4"
+                operating-system:
+                    - "ubuntu-latest"
+
+        steps:
+            - name: "Checkout"
+              uses: "actions/checkout@v2"
+
+            - name: "Install PHP"
+              uses: "shivammathur/setup-php@v2"
+              with:
+                  tools: "phing"
+                  coverage: "none"
+                  php-version: "${{ matrix.php-version }}"
+                  ini-values: memory_limit=-1, phar.readonly=0
+
+            - name: "Cache dependencies"
+              uses: "actions/cache@v2"
+              with:
+                  path: |
+                      ~/.composer/cache
+                      vendor
+                  key: "php-${{ matrix.php-version }}-${{ matrix.dependencies }}"
+                  restore-keys: "php-${{ matrix.php-version }}-${{ matrix.dependencies }}"
+
+            - name: "Install lowest dependencies"
+              if: ${{ matrix.dependencies == 'lowest' }}
+              run: "composer update --prefer-lowest --no-interaction --no-progress --no-suggest"
+
+            - name: "Install highest dependencies"
+              if: ${{ matrix.dependencies == 'highest' }}
+              run: "composer update --no-interaction --no-progress --no-suggest"
+
+            - name: "Install locked dependencies"
+              if: ${{ matrix.dependencies == 'locked' }}
+              run: "composer install --no-interaction --no-progress --no-suggest"
+
+            - name: "build via phing"
+              run: "phing"
+
+            - name: "Upload phar file artifact"
```

Comment:
Given that this runs `on: [pull_request, branch]`, where is this artifact to be found? Just attached to builds?

Reply:
Yes, the phar is available and attached to the build, for this PR it would be here: https://github.com/maglnet/ComposerRequireChecker/actions/runs/368166959

Reply:
Works for me, but not really a stable distribution channel.

If this is only for testing, it should be fine :+1:

Reply:
Yes only for testing new features / bugfixes as a phar.
And as a nice on top: It's testing the phar building process itself - since it the phar now getting build for every PR :)